### PR TITLE
Flex Message Update 1

### PR DESCRIPTION
**Pull request by okue**
#204 

- add new fields for Flex message
- add tests
- add an example using a new spec of Flex message

### CODE REVIEW

File: linebot/models/flex_message.py [hunk 1]
```diff
@@ -420,6 +555,42 @@ def __init__(self, size=None, **kwargs):
         self.size = size
 
 
+class SpanComponent(FlexComponent):
+    """SpanComponent.
+
+    https://developers.line.me/en/docs/messaging-api/reference/#span-component
```
Comment (tkgauri) [inline linebot/models/flex_message.py]: https://developers.line.biz/en/reference/messaging-api/#span

File: examples/flask-kitchensink/app.py [hunk 2]
```diff
@@ -332,6 +333,137 @@ def handle_text_message(event):
             event.reply_token,
             message
         )
+    elif text == 'flex_update_1':
+        bubble_string = """
```
Comment (cryeo) [inline examples/flask-kitchensink/app.py]: [ask]
Is there any reason to define as raw JSON?

[imo]
From the viewpoint of providing examples, it is better to define by using models SDK provides like the following:

```python
        bubble = BubbleContainer(
            body=BoxComponent(
                layout="vertical",
                padding_all="0px",
                contents=[
                    ImageComponent(
                        url="https://line-objects-dev.com/flex/bg/eiji-k-1360395-unsplash.jpg",
                        position="relative",
                        size="full",
                        aspect_mode="cover",
                        aspect_ratio="1:1",
                        gravity="center"
                    ),
                    BoxComponent(
                        layout="horizontal",
                        position="absolute",
                        padding_all="20px",
                        offset_bottom="0px",
                        offset_start="0px",
                        offset_end="0px",
                        background_color="#00000099",
                        contents=[
                            BoxComponent(
                                layout="vertical",
                                contents=[
                                    TextComponent(
                                        text="Brown Hotel",
                                        weight="bold",
                                        size="xl",
                                        color="#ffffff"
                                    ),
                                    BoxComponent(
                                        layout="baseline",
                                        margin="md",
                                        contents=[
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gray_star_28.png"
                                            ),
                                            TextComponent(
                                                size="sm",
                                                text="4.0",
                                                color="#d6d6d6",
                                                margin="md",
                                                flex=0,
                                            )
                                        ]
                                    )
                                ]
                            ),
                            BoxComponent(
                                layout="vertical",
                                contents=[
                                    TextComponent(
                                        text="¥62,000",
                                        color="#a9a9a9",
                                        decoration="line-through",
                                        align="end"
                                    ),
                                    TextComponent(
                                        text="¥42,000",
                                        color="#ebebeb",
                                        size="xl",
                                        align="end"
                                    )
                                ]
                            )
                        ]
                    ),
                    BoxComponent(
                        layout="vertical",
                        position="absolute",
                        background_color="#ff2600",
                        corner_radius="20px",
                        padding_all="5px",
                        offset_top="10px",
                        offset_end="10px",
                        padding_start="10px",
                        padding_end="10px",
                        contents=[
                            TextComponent(
                                text="SALE",
                                color="#ffffff"
                            )
                        ]
                    )
                ]
            )
        )
```

File: linebot/models/flex_message.py [hunk 3]
```diff
@@ -276,6 +356,11 @@ def __init__(self, action=None, flex=None, margin=None, height=None, style=None,
         :param str color: Character color when the style property is link.
             Background color when the style property is primary or secondary.
             Use a hexadecimal color code
+        :param str position: Reference position for placing this box
```
Comment (cryeo) [inline linebot/models/flex_message.py]: [nits]
The order of parameters in docstring is not the same as that in the method definition

File: linebot/models/flex_message.py [hunk 4]
```diff
@@ -325,26 +426,50 @@ def __init__(self, url=None, margin=None, size=None, aspect_ratio=None, **kwargs
             and the previous component in the parent box
         :param str size: Maximum size of the icon width
         :param str aspect_ratio: Aspect ratio of the icon
+        :param str position: Reference position for placing this box
```
Comment (cryeo) [inline linebot/models/flex_message.py]: [nits]
The order of parameters in docstring is not the same as that in the method definition

File: linebot/models/flex_message.py [hunk 5]
```diff
@@ -362,13 +487,23 @@ def __init__(self, url=None, flex=None, margin=None, align=None, gravity=None, s
         :param str background_color: Background color of the image. Use a hexadecimal color code.
         :param action: Action performed when this image is tapped
         :type action: list[T <= :py:class:`linebot.models.actions.Action`]
+        :param str position: Reference position for placing this box
```
Comment (cryeo) [inline linebot/models/flex_message.py]: [nits]
The order of parameters in docstring is not the same as that in the method definition

File: examples/flask-kitchensink/app.py (hunk 2 above)
Comment (cryeo) [reply examples/flask-kitchensink/app.py]: [memo]
https://github.com/line/line-bot-sdk-python/pull/206#pullrequestreview-282329512

File: linebot/models/flex_message.py (hunk 3 above)
Comment (okue) [reply linebot/models/flex_message.py]: fixed

**Pull request merged**